rcmd/install.go

```diff
@@ -380,17 +381,30 @@ func InstallPackagePlan(
 	es ExecSettings,
 	ncpu int,
 ) error {
+
+	//var successCounter uint64
 	wg := sync.WaitGroup{}
+
 	startTime := time.Now()
+
 	// for now this will only be updated in the Update function
 	// however if it may be concurrently accessed should consider a syncmap implementation
 	installedPkgs := make(map[string]bool)
 	// if packages ID'd as ready to install signal so can push them only the queue
 	shouldInstall := make(chan string)
 	anyFailed := false
+
+	toInstall := plan.GetNumPackagesToInstall(viper.GetBool("update"))
```

Comment:
see above comment

Reply:
Agreed. The only reason I didn't was because it felt odd to me to add an "update" parameter to this function when it's only used for logging.

I think what I'll try to do is store the "update" boolean in the InstallPlan, then this method can just read from there instead of taking it as an argument.